api/v1/fluxinstance_types.go

```diff
@@ -83,6 +83,13 @@ type Distribution struct {
 	// to use for pulling images.
 	// +optional
 	ImagePullSecret string `json:"imagePullSecret,omitempty"`
+
+	// Artifact is the URL to the OCI artifact containing
+	// the latest Kubernetes manifests for the distribution,
+	// e.g. 'oci://ghcr.io/controlplaneio-fluxcd/flux-operator-manifests:latest'.
+	// +kubebuilder:validation:Pattern="^oci://.*$"
+	// +optional
+	Artifact string `json:"artifact,omitempty"`
```

Comment:
can we name it maybe `artifactURL`?

Reply:
I did consider this but given that we use `registry` and not `registryURL`, for consistency I chose to not have the URL suffix.